### ISSUE TITLE
Trying to build throws tons of "header not found" errors

### ISSUE THREAD

**Issue by Linux-cpp-lisp** [rjyo/libid3-ios#3]
I'm trying to integrate this into a project, and it's not working. Every time I try to build, all of the id3lib files are complaining that they can't find the various id3lib headers. 

If it would have any effect, I dragged the project into the frameworks group, and downloaded it as a git submodule. 

The same thing happens when I try to build the Example target.

**Comment by Vienta** [comment]
@Linux-cpp-lisp   Did you solved this? I have the same issue

**Comment by Linux-cpp-lisp** [comment]
@Vienta : Nope. Sorry. I gave up on ID3 tags for that project altogether, and just sent the data as a URL alongside the MP3.
